### PR TITLE
Remove the last purple stuff

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

File: tools/CMakeLists.txt [hunk 1]
```diff
@@ -23,45 +6,35 @@ add_definitions(-DEQ_SYSTEM_INCLUDES) # get GL headers
 list(APPEND CPPCHECK_EXTRA_ARGS -I${PROJECT_SOURCE_DIR}/examples/include
   -I${PROJECT_SOURCE_DIR}/examples)
 
-if(WIN32)
-  eq_add_tool(affinityCheck SOURCES affinityCheck/affinityCheck.cpp
-    LINK_LIBRARIES ${GLEW_LIBRARY} ${OPENGL_gl_LIBRARY}
-    )
-endif()
+set(EQTHREADAFFINITY_SOURCES threadAffinity/threadAffinity.cpp)
+set(EQTHREADAFFINITY_LINK_LIBRARIES Equalizer EqualizerServer)
```
Comment (tribal-tec) [inline tools/CMakeLists.txt]: ${GLEW_LIBRARY} ${OPENGL_gl_LIBRARY} is still enough

Comment (eile) [inline tools/CMakeLists.txt]: Merge error - fixed

**Comment by eile** [comment]
retest this please

**Comment by tribal-tec** [comment]
+1

**Comment by eile** [comment]
retest this please

**Pull request merged**